client/js/socket-events/configuration.js

```diff
@@ -7,10 +7,9 @@ const options = require("../options");
 const webpush = require("../webpush");
 
 socket.on("configuration", function(data) {
-	if (!options.initialize) {
+	if (options.initialized) {
```

Comment:
Hmmm, why this change?

Reply:
Before it wasn't a boolean and checked something else.

Reply:
To be a bit more clear, before it set `module.exports.initialize` to `null` which works (but a bit overkill) but also resulted in the negative check for initialize which is slightly confusing as it is more of a `true` situation as options are in fact initialized. 

tl;dr readability of code mostly.

Reply:
Aha, I see